src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -36,24 +38,21 @@ const useStyles = makeStyles((theme: Theme) =>
     },
     textField: {
       padding: "0.2rem 1rem",
-      cursor: "none"
+      cursor: "none",
+      width: "100%"
     },
     input: {
       cursor: "pointer",
       borderBottom: "1px dotted black",
       paddingBottom: "0.25rem"
     },
-    underline: {
-      "&&&:before": {
-        borderBottom: "none"
-      },
-      "&&:after": {
-        borderBottom: "none"
-      }
-    },
     sourceIcon:{
       marginRight: "0.2rem",
       fill: "#8080809c"
+    },
+    dictionaryIcon:{
+      marginRight: "0.2rem",
+      fill: "#8080809c"
```

Comment:
We don't really need a separate `dictionaryIcon` class... This can be shared with `sourceIcon` (at least until we develop a need to style the icons differently)